src/server/auth.ts

```diff
@@ -57,38 +57,65 @@ const getFirstNameAndLastName = (
   };
 };
 
+function generateRandomString(length: number): string {
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+  let result = '';
+  const charactersLength = characters.length;
+
+  for (let i = 0; i < length; i++) {
```

Comment:
See if you can use `.map` or `.reduce` instead.

Also we should probably use a library or built-in language features for this `generateRandomString` function otherwise it'll end up being a utility and may have to be maintained. You can define randomization seeds/params with regexes.